lib/travis/api/v3/service_index.rb

```diff
@@ -41,6 +41,7 @@ def all_resources
         all = routes.resources + [
           Routes::Resource.new(:broadcast), # dummy as there are only broadcasts routes right now
           Routes::Resource.new(:commit),    # dummy as commits can only be embedded
+          Routes::Resource.new(:message),   # dummy as there is only a message route right now
```

Comment:
- only a message
+ only a messages

Right?

Reply:
Messages has a service so is picked up by the service_index

Reply:
yes